assets/js/modules/tagmanager/components/common/FormInstructions.js

```diff
@@ -43,7 +43,7 @@ export default function FormInstructions() {
 
 	// Multiple property IDs implies secondary AMP where selected containers don't reference the same Analytics property ID.
 	if ( hasMultipleAnalyticsPropertyIDs ) {
-		const message = __( 'Looks like you’re already using Google Analytics within your Google Tag Manager configurations. However, the configured Analytics tags reference different property IDs. You need to configure the same Analytics property in both containers.', 'google-site-kit' );
+		const message = __( 'Error: Looks like you’re already using Google Analytics within your Google Tag Manager configurations. However, the configured Analytics tags reference different property IDs, or the setup process was not fully completed. You need to configure the same Analytics property in both containers.', 'google-site-kit' );
```

Comment:
The `ErrorText` component, which renders this message, [already prepends "Error: "](https://github.com/google/site-kit-wp/blob/f30bc93d58c3d74467073cc4793073d62d4d600b/assets/js/components/ErrorText.js#L42) to the string here, so adding this results in this text being display:

<img width="1020" alt="Screenshot 2021-06-22 at 20 09 50" src="https://user-images.githubusercontent.com/90871/122985217-d01bc200-d395-11eb-9619-b7e094bde036.png">

I'm going to remove that text, but please make sure to test out the changes locally before moving things to Code Review. This error can be seen in the story at: https://google.github.io/site-kit-wp/storybook/pull/3492/?path=/story/tag-manager-module-setup-secondary-amp--multiple-property-ids-analytics-active